app/reducers/forms.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 
+import { createSelector } from 'reselect';
```

Comment:
It looks amazing! Is this method used anywhere specific? Can't seem to spot it...

Reply:
Oops. forgot to remove this when I decided not to create a selector.
Nice catch :1st_place_medal: